### ISSUE TITLE
PHP 8.2 :) :)

### ISSUE THREAD

**Issue by PhilETaylor** [scheb/2fa#162]
```
  - scheb/2fa[v6.2.0, ..., 6.x-dev] require php ~8.0.0 || ~8.1.0 -> your php version (8.2.0-dev) does not satisfy that requirement.
```

I know you know :) but might be helpful now to bump this as 8.2.0RC1 is live now :)

**Comment by scheb** [comment]
Branch [php8.2](https://github.com/scheb/2fa/tree/php8.2) is now available to play around with PHP 8.2. Will merge to main branch and create a proper release, once PHP 8.2 is officially released.

Btw. thanks for your donation, much appreciated :)

**Comment by PhilETaylor** [comment]
Awesome - thanks. I'll pin to that. 

> now available to play around with PHP 8.2.

You mean I should not be running PHP 8.2 in production on a SaaS used by 1000s of paying customers?, like a mad man, um, nevermind haha. https://twitter.com/mysitesguru/status/1566080219374665729

> Btw. thanks for your donation, much appreciated :)

You're welcome.

**Comment by scheb** [comment]
Well, I guessed that you're doing it, since you like living on the edge ;)

**Comment by PhilETaylor** [comment]
Hey Alexa, "create a cron job to run composer update every 10 mins on production" LMAO.

Seriously though, its a testament to your great work that it "just works ™️" on PHP 8.2 with no deprecated messages or errors - we are both winning here!

**Comment by ausi** [comment]
> Will merge to main branch and create a proper release, once PHP 8.2 is officially released.

PHP 8.2.0 is already tagged: https://github.com/php/php-src/releases/tag/php-8.2.0
And the official release is planned for today.

_(this is meant as a reminder, not a demand ☺️)_

Thank you very much for all your work!

**Comment by scheb** [comment]
👉 Released v6.4.0 including PHP 8.2 support
